lib/config/presets/internal/group.ts

```diff
@@ -572,6 +572,19 @@ const staticGroups = {
       },
     ],
   },
+  jestPlusTSJest: {
+    description: 'Group major updates for jest and ts-jest together',
```

Comment:
```suggestion
    description: 'Group all major jest packages together',
```

Reply:
```suggestion
    description: 'Group major updates for ts-jest to jest monorepo',
```

Something like this?